src/plugins/intel_gna/src/ops/util/util.hpp

```diff
@@ -274,6 +279,115 @@ static bool has_32bit_output(const std::shared_ptr<ngraph::Node>& node) {
 inline bool has_32bit_input(const std::shared_ptr<ngraph::Node>& node) {
     return is_activation(node) || is_pooling(node);
 }
+
+static bool is_op_supported(const std::shared_ptr<ov::Node>& node,
+                            const std::string& gnaCompileTarget,
+                            const GNAPluginNS::Config config,
+                            bool raise_exception = false) {
+    if (ngraph::op::is_parameter(node)) {
+        const std::set<ov::element::Type> supported = { ov::element::u8, ov::element::i16, ov::element::f32 };
```

Comment:
We already checks input precisions while loading network, at least we can reuse supported list from there

Reply:
> We already check input precisions while loading network, at least we can reuse supported list from there

It is legacy one
```
        if (network_input_precision != InferenceEngine::Precision::FP32 &&
            network_input_precision != InferenceEngine::Precision::I16 &&
            network_input_precision != InferenceEngine::Precision::U8) {
            errMessage = "The plugin does not support input precision with " +
                         std::string(network_input_precision.name()) +
                         " format. Supported  input precisions FP32, I16, U8\n";
            return false;
        }
```

Reply:
I don't agree, 
It only uses some legacy types, but the logic is there, so I would not introduce the same logic, instead I would share as much as possible